samples/sample.js

```diff
@@ -1,23 +1,26 @@
 'use strict';
 
 const { diag, DiagConsoleLogger, DiagLogLevel } = require('@opentelemetry/api');
-const { MeterProvider } = require('@opentelemetry/sdk-metrics-base');
+const { Resource } = require('@opentelemetry/resources');
+const { MeterProvider, PeriodicExportingMetricReader, ConsoleMetricExporter, AggregationTemporality } = require('@opentelemetry/sdk-metrics-base');
 const { DynatraceMetricExporter } = require('..');
 
 diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.ALL)
 
-const exporter = new DynatraceMetricExporter({
+let exporter = new DynatraceMetricExporter({
   prefix: 'sample', // optional
 
   // If no OneAgent is available locally, export directly to the Dynatrace server:
   // url: 'https://myenv123.live.dynatrace.com/api/v2/metrics/ingest',
   // APIToken: '<load API token from secure location such as env or config file>'
```

Comment:
I am not sure how strict JS usually is with these things, but should we use `apiToken` to have consistent camelCase?

Reply:
Yeah normally you would but this is already released so I wouldn't want to break the API

Reply:
Thinking if we should still do this, though. :thinking: 
Users will have to change quite a few things going form the previous SDK to the new one, and we're still in `0.x` :thinking:

Reply:
Checked with armin and agreed to do it with a backwards compatible deprecation note